### PR TITLE
L3: functions

### PR DESCRIPTION
**Pull request by Brandonz07**

### CODE REVIEW

File: src/br/js/L3-functions/BeanCounting.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  let count = 0;
+  for (let i = 0; i < a.length; i++) {
+    if (a[i] === "B") {
+      count++;
+    }
+  }
+  return count;
+}
+
+function countChar(a, b) {
```
Comment (carin91) [inline src/br/js/L3-functions/BeanCounting.js:11]: It is recommended to use names for the variables that could be understandable by everyone. Change The names for all of the arguments of the components.

File: src/br/js/L3-functions/BeanCounting.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+function countBs(a) {
```
Comment (carin91) [inline src/br/js/L3-functions/BeanCounting.js:1]: It is recommended to use names for the variables that could be understandable by everyone. Change The names for all of the arguments of the components.

File: src/br/js/L3-functions/BeanCounting.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+function countChar(a, b) {
+  let count = 0;
+  for (let i = 0; i < a.length; i++) {
+    if (a[i] === b) {
+      count++;
+    }
+  }
+  return count;
+}
+
```
Comment (carin91) [inline src/br/js/L3-functions/BeanCounting.js:20]: There is pending 1 activity: Rewrite countBs to make use of this new function.

File: src/br/js/L3-functions/Minimum.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+function min(a, b) {
+  if (a < b) {
+    return a;
+  } else if (a > b) {
```
Comment (carin91) [inline src/br/js/L3-functions/Minimum.js:4]: avoid unnecessary calculations.

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/br/js/L3-functions/BeanCounting.js (hunk 1 above)
Comment (Brandonz07) [reply src/br/js/L3-functions/BeanCounting.js]: change variable names

File: src/br/js/L3-functions/BeanCounting.js (hunk 2 above)
Comment (Brandonz07) [reply src/br/js/L3-functions/BeanCounting.js]: change variable names

File: src/br/js/L3-functions/BeanCounting.js (hunk 3 above)
Comment (Brandonz07) [reply src/br/js/L3-functions/BeanCounting.js:14]: I rewrote counBs

File: src/br/js/L3-functions/Minimum.js (hunk 4 above)
Comment (Brandonz07) [reply src/br/js/L3-functions/Minimum.js]: use a ternary operator

**Review by carin91 — APPROVED** [review]

**Pull request merged**